CMakeLists.txt

```diff
@@ -254,6 +254,7 @@ if(VELOX_ENABLE_ABFS)
   endif()
   # files-datalake is built on blobs
   find_package(azure-storage-files-datalake-cpp CONFIG REQUIRED)
+  find_package(azure-identity-cpp CONFIG REQUIRED)
```

Comment:
```suggestion
```
Add this in the PR that makes use of it (fails CI as the image isn't updated yet)

Reply:
Sure, updated.